documentation/book/appendix_metrics.adoc

```diff
@@ -37,64 +37,154 @@ endif::Kubernetes[]
+
+* A `ClusterRole` that grants permissions to read the Prometheus health endpoints of the {OpenShiftName} system, including cAdvisor and the kubelet for container metrics. The Prometheus server configuration uses the {OpenShiftName} service discovery feature in order to discover the pods in the cluster from which it gets metrics.  For this feature to work correctly, the service account used for running the Prometheus service pod must have access to the API server so it can retrieve the pod list.
 * A `ServiceAccount` for the Prometheus pods to run under.
 * A `ClusterRoleBinding` which binds the aforementioned `ClusterRole` to the `ServiceAccount`.
-* A `Deployment` to manage the actual Prometheus server pod.
-* A `ConfigMap` to manage the configuration of Prometheus Server.
-* A `Service` to provide an easy to reference hostname for other services to connect to Prometheus server (such as Grafana).
+* A `Deployment` to manage the Prometheus Operator pod.
+* A `ServiceMonitor` to manage the configuration of the Prometheus pod.
+* A `Secret` to manage additional Prometheus settings.
+* A `PrometheusRule` to manage alert rules for the Prometheus pod.
+* A `Prometheus` to manage the configuration of the Prometheus pod.
 
-Prometheus also provides an alerting system through the link:https://prometheus.io/docs/alerting/alertmanager/[alert manager] component.
-In order to enable alerting, the provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml[`alerting-rules.yaml`] file describes a `ConfigMap` resource which defines sample alerting rules on Kafka and Zookeeper metrics.
-When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the alert manager which then uses the configured notification methods to notify the user.
+Prometheus also provides an alerting system through the link:https://prometheus.io/docs/alerting/alertmanager/[Alertmanager] component.
+To enable alerting, the provided link:https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/install/prometheus-rules.yaml[`prometheus-rules.yaml`] file describes a `PrometheusRule` resource that defines sample alerting rules for Kafka and Zookeeper metrics.
+When an alert condition is evaluated as true on the Prometheus server, it sends the alert data to the Alertmanager which then uses the configured notification methods to notify the user.
 
-More information about how to set up alerting rules https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[here].
+For more information about setting up alerting rules, see https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/[Alerting Rules] in the Prometheus documentation.
 
-==== Deploying on {OpenShiftName}
+==== Deploying the Prometheus Operator on {KubernetesName}
```

Comment:
Does this differ from OpenShift in anything else then `oc` versus `kubectl`? If not maybe we can join it into a single chapter.

Reply:
It does differ also in need to remove a `SecurityContext`

Reply:
The security context stuff is written in very generic form ... _When needed ..._ ... I wonder if that is enough to have seprate chapters for OpenShift and Kube.